src/app/(admin)/dashboard/page.tsx

```diff
@@ -1,51 +1,58 @@
 import DashboardCard from '@/components/dashboard/DashboardCard'
 import PendingAdminDeals from './PendingDeals'
 import DealListByViews from '@/components/dashboard/metrics/DealListByViews'
-import Link from 'next/link'
 import ReportedDeals from '@/components/dashboard/reports/ReportedDeals'
+import FeaturedAdminDeals from './FeaturedDeals'
+import SubscriberList from '@/components/subscriber/SubscriberList'
 
 export default async function RootDashboardPage() {
   return (
     <section>
       <h1 className="mb-12 text-center text-5xl text-white">Dashboard</h1>
       <div className="grid grid-cols-2 gap-8">
-        <DashboardCard heading="Pending Deals">
+        <DashboardCard
+          heading="Pending Deals"
+          href="/dashboard/deals/pending"
+          linkText="View All Pending Deals"
+        >
           <div className="mb-6">
```

Comment:
nit: Instead of adding `mb-6` to these divs I believe you can add `space-y-6` to the `DashboardCard` component to apply to all children of it.